pkg/edgedictionaryitem/list.go

```diff
@@ -0,0 +1,54 @@
+package edgedictionaryitem
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/fastly/cli/pkg/common"
+	"github.com/fastly/cli/pkg/compute/manifest"
+	"github.com/fastly/cli/pkg/config"
+	"github.com/fastly/cli/pkg/errors"
+	"github.com/fastly/cli/pkg/text"
+	"github.com/fastly/go-fastly/fastly"
+)
+
+// ListCommand calls the Fastly API to describe a service.
+type ListCommand struct {
+	common.Base
+	manifest manifest.Data
+	Input    fastly.ListDictionaryItemsInput
+}
+
+// NewListCommand returns a usable command registered under the parent.
+func NewListCommand(parent common.Registerer, globals *config.Data) *ListCommand {
+	var c ListCommand
+	c.Globals = globals
+	c.manifest.File.Read(manifest.Filename)
+	c.CmdClause = parent.Command("list", "List items in a Fastly edge dictionary")
+	c.CmdClause.Flag("service-id", "Service ID").Short('s').StringVar(&c.manifest.Flag.ServiceID)
+	c.CmdClause.Flag("dictionary-id", "Dictionary ID").Required().StringVar(&c.Input.Dictionary)
+	return &c
+}
+
+// Exec invokes the application logic for the command.
+func (c *ListCommand) Exec(in io.Reader, out io.Writer) error {
+	serviceID, source := c.manifest.ServiceID()
+	if source == manifest.SourceUndefined {
+		return errors.ErrNoServiceID
+	}
+	c.Input.Service = serviceID
+
+	dictionaries, err := c.Globals.Client.ListDictionaryItems(&c.Input)
+	if err != nil {
+		return err
+	}
+
+	fmt.Fprintf(out, "Service ID: %s\n", c.Input.Service)
```

Comment:
This (and other instances) should be replaced with the `text` helper package equivalent I believe. 

/cc @phamann

Reply:
Good call. I was following an existing pattern here, but it never hurts to have more code correct when the refactor happens later.

Reply:
Awesome, thank you!

Reply:
I fixed this in this PR. I started to fix it in other places but there is a more fundamental change required in several the 900+ uses that needs to be addressed in that text.Output trims whitespace and several of the fmt.Fprintf instances have preceding tabs for pretty printing.

Reply:
@benzvan no problem, if it's a long standing inconsistency then I'll try to address it at some later point in time 🙂  as long as new code added is avoiding the use of fmt in favour of a text package functional equivalent, then that's fine 👍🏻